diff --git a/target/linux/ipq806x/files-4.9/arch/arm/boot/dts/qcom-ipq8064-ea8500.dts b/target/linux/ipq806x/files-4.9/arch/arm/boot/dts/qcom-ipq8064-ea8500.dts
index 761fa43..7850ac3 100644
--- a/target/linux/ipq806x/files-4.9/arch/arm/boot/dts/qcom-ipq8064-ea8500.dts
+++ b/target/linux/ipq806x/files-4.9/arch/arm/boot/dts/qcom-ipq8064-ea8500.dts
@@ -157,19 +157,13 @@
 
 		pcie0: pci@1b500000 {
 			status = "ok";
-			phy-tx0-term-offset = <7>;
 		};
 
 		pcie1: pci@1b700000 {
 			status = "ok";
-			phy-tx0-term-offset = <7>;
+			force_gen1 = <1>;
 		};
-		
-		pcie2: pci@1b900000 {
-			status = "ok";
-			phy-tx0-term-offset = <7>;
-		};
-		
+
 		nand@1ac00000 {
 			status = "ok";
 
